src/components/AuthenticatedContributeMeasurementForm.tsx

```diff
@@ -20,19 +20,21 @@ import {
 import { FormTextInput, RadioButtonComponent } from "./form-components";
 import { SignalTimer } from "./SignalTimer";
 import { RawOSMCrossing, getOSMCrossings } from "../api/overpass";
+import { isNodeValid } from "../api/osm"
 const MAPBOX_TOKEN =
   "pk.eyJ1IjoiamFrZWMiLCJhIjoiY2tkaHplNGhjMDAyMDJybW4ybmRqbTBmMyJ9.AR_fnEuka8-cFb4Snp3upw";
 
 export interface AuthenticatedFormProps {
   session: Session;
+  nodeId: string
```

Comment:
As per lower comment perhaps this could be `undefined | string`

Reply:
yeap, that'd done